src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -166,13 +164,18 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     firstScheduleDate,
   ]);
 
+  const relatedVenueRooms =
+    relatedVenues
+      .flatMap((venue) => venue.rooms || emptyArrayObj)
+      .filter((x) => !!x) || emptyArrayObj;
+
   const roomList = scheduleNG.daysEvents.map((el) => {
     const [roomData] =
-      currentVenue?.rooms?.filter((room) => room.title === el?.room) || [];
+      relatedVenueRooms.filter((room) => room?.title === el?.room) ||
+      emptyArrayObj;
```

Comment:
```suggestion
      relatedVenueRooms.filter((room) => room?.title === el?.room)
```
U'll always have an empty array based on the above logic

Reply:
can't agree, it's either we filter out rooms that don't match and have 1 room that matches or an empty array

Reply:
removed in 7cc46b5 anyway